### DISCUSSION TITLE
Internal Labels for Array Items too (not just Blocks)

### DISCUSSION THREAD

**Discussion by christian-reichart** [payloadcms/payload#1884] [Feature Requests & Ideas]
Just like with blocks, it would be helpful to assign a label to each entry in an array list.

Blocks:
![grafik](https://user-images.githubusercontent.com/9932248/212637298-8d19c08f-4fc2-4f64-9552-d3d9c3f16caa.png)

Arrays:
![grafik](https://user-images.githubusercontent.com/9932248/212637394-897a756a-b144-49cd-a7e5-5047c1a308f5.png)

**Comment by colkas** [comment]
This would be very helpful :)
